### PR TITLE
Add similairty score model

### PR DESCRIPTION
**Pull request by anushkabhatcode**
This notebook is an exploration of various ways a similarity score can be generated between a resume and a job description. Many ways were explored but TF-IDF performed well from the perspective of human review. Hence, this notebook is being committed to the repo as opposed to others. More changes are expected as the project progresses.

### CODE REVIEW

**Comment by zeshengchen** [comment]
Please add the "description" to this pull request.

Moreover, if the code refers to the existing open-source project or was generated by an AI tool, please indicate it at the beginning of the Jupyter Notebook.

File: Models/ResumeBuilder_SimilarityScore.ipynb
Comment (pktpaulie) [inline Models/ResumeBuilder_SimilarityScore.ipynb:1]: Great job implementing the similarity score. I get pretty low weighted scores of 0.018 for a description that's not close to the resume and 0.2 for a description that is somewhat close to the resume.

**Review by pktpaulie — APPROVED** [review]

**Review by Kavyan17 — APPROVED** [review]
Combination of TF-IDF and SBERT is effective.

**Review by KarthikBalajiB — APPROVED** [review]
Looks good.

**Comment by Anjalipolasani** [comment]
looks great

**Comment by zeshengchen** [comment]
> looks great

Pleas approve it!

**Review by Anjalipolasani — APPROVED** [review]
great implementation

**Pull request merged**